xbmc/Application.cpp

```diff
@@ -3173,8 +3173,7 @@ void CApplication::OnPlayBackStarted(const CFileItem &file)
   CLog::LogF(LOGDEBUG,"CApplication::OnPlayBackStarted");
 
   // check if VideoPlayer should set file item stream details from its current streams
-  if (file.GetProperty("get_stream_details_from_player").asBoolean())
-    m_appPlayer.SetUpdateStreamDetails();
+  m_appPlayer.SetUpdateStreamDetails();
```

Comment:
An alternative could be to only update (or store) details when we don't have any yet.

Reply:
Sure. Is ```if (file.GetProperty("get_stream_details_from_player").asBoolean() || file.IsDiscImage() ||
      file.IsDVDFile() || file.IsBDFile())``` enough or does it need some more ?

Reply:
I think it also requires IsDVD/IsOnDVD/IsBluray, not sure though.

Reply:
I'd go with only updating or storing details when we don't have any but this would mean that for dvds with multiple audio tracks, eg 2.0 & 5.1 that the details would be set on first play.  If I played it in stereo first then it would always show as that, even if I later played it in surround. Personally I'd prefer it to update to what it is currently but I'm fine with not doing so if that's what the team wants.

Reply:
Good point, I'm with you then.

Please also have a look at the comments in https://github.com/xbmc/xbmc/pull/15589 as to why ALWAYS updating (Without a condition) stream details was causing problems.

Reply:
OK, done.